virt-manager.rb

```diff
@@ -3,57 +3,62 @@ class VirtManager < Formula
 
   desc "App for managing virtual machines"
   homepage "https://virt-manager.org/"
-  url "https://virt-manager.org/download/sources/virt-manager/virt-manager-2.2.1.tar.gz"
-  sha256 "cfd88d66e834513e067b4d3501217e21352fadb673103bacb9e646da9f029a1b"
-  revision 3
+  url "https://releases.pagure.org/virt-manager/virt-manager-3.2.0.tar.gz"
```

Comment:
why are you not using https://virt-manager.org/download/sources/  ?

Reply:
I guess I opened https://virt-manager.org/download/sources/virt-manager/ which redirects to release.pagure.org :)